src/api/overlays/Overlay.js

```diff
@@ -28,6 +28,8 @@ class Overlay extends EventEmitter {
             template: '<overlay-component></overlay-component>'
         });
 
+        this.notDismissable = options.notDismissable;
```

Comment:
I have a preference for using positive rather than negative logic, I think it's slightly easier to parse by humans. 
eg. `dismissable` rather than `notDismissable`. It also leads to less funky boolean operations, eg.

```javascript
        if (lastOverlay && !lastOverlay.notDismissable) {
            lastOverlay.dismiss();
        }
```
might be a little easier to mentally process if it was rewritten as 
```javascript
        if (lastOverlay && lastOverlay.dismissable) {
            lastOverlay.dismiss();
        }
```